keymanapp/keyman
# chore(windows): remove keyman_forcekeyboard

Fixes: #8646

## User Testing


# TEST_SELECT_DIFFERENT_KBDS
Install two or three keyboards such as  Tamil 99 keyboard and Khmer Angkor keyboard

*  Open Notepad, select the Tamil Keyboard 
* Type in some characters and ensure they are output
* Select the Khmer_Angkor Keyboard 
* Type in some characters and ensure they are output
* Verify that typing produces the expected output
* Select Tamil Keyboard again
* Verify that typing produces the expected output
